### PR TITLE
Fixed #54: Clarified use of PTS for buffered

### LINKED ISSUE #54: CTS vs DTS, which is correct?
I could not find any information in the spec about whether `buffered` should give the DTS or the CTS of the appended frames.  Firefox and Safari show CTS, while Chrome and IE/Edge show DTS.  Is one or the other to-spec, or are both valid implementations that applications must deal with?

### PR DESCRIPTION
**Pull request by jdsmith3000**
Added note to section 3.3 Track Buffers clarifying use of PTS, frame
durations and coded frame group start times in track buffers.

### CODE REVIEW

File: media-source-respec.html [hunk 1]
```diff
@@ -2723,10 +2726,19 @@
...
+            <td>April 28 2016</td>
             <td>
               <ul>
-                <li>Issue 17 &amp; 18 - Generalize the detach MediaSource algorithm.</li>
+                <li>Issue 54 - Clarified use of Presentation Time Stamps (PTS) for <a def-id="buffered"></a>.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr>
+            <td><a href="https://rawgit.com/w3c/media-source/365acf8c3c50aa253439782894d5ee9492b81039/index.html">April 15 2016</td>
```
Comment (wolenetz) [inline media-source-respec.html]: This URL points to a version of the spec that is timestamped Nov 12, 2015. You probably want to instead reference https://rawgit.com/w3c/media-source/a0bf0e68c8e5e53a9450abe94b83a9c3a6f8f13e/index.html 

Please also update media-source-respec.html's publishDate and regenerate index.html.

**Comment by wolenetz** [comment]
Just a couple minor nits (https://github.com/w3c/media-source/pull/70#discussion-diff-61495954). Let me know when this is ready to merge and I can do final review and merge it if it l*tm.
It might be good to define PTS "Presentation Time Stamps (PTS)" in the note, too, not just in the change history, but I'm fine if you don't go that far :)
Thanks!

**Comment by jdsmith3000** [comment]
@Wolenetz I've picked up your requested changes.

**Comment by wolenetz** [comment]
@jdsmith3000 changes look good. I'll go ahead and resolve the merge conflicts and merge this now.

**Comment by wolenetz** [comment]
@plh keeps landing other stuff upstream. I'll rebase again and try merging yet again :)

**Comment by plehegar** [comment]
oops, sorry. was fixing various metadata for the /TR update. I'm done now.

**Comment by wolenetz** [comment]
Merged as f6be5bc and cc7e3e8

**Comment by wolenetz** [comment]
@plh no worries - thanks for your work!

**Pull request closed without merging**